lib/opentelemetry/trace/exporter/otlp.lua

```diff
@@ -13,10 +14,16 @@ local mt = {
     __index = _M
 }
 
-function _M.new(http_client, timeout_ms)
+function _M.new(http_client, timeout_ms, circuit_reset_timeout_ms, circuit_open_threshold)
     local self = {
         client = http_client,
         timeout_ms = timeout_ms or DEFAULT_TIMEOUT_MS,
+        failure_count = 0,
+        success_count = 0,
```

Comment:
failure_count and. success_count not used?

Reply:
You are right. Left over from older version. Thank you! d98404146ffe495e7d24f0c00cba1ce51e229f47